docs/user-guide/configuration.rst

```diff
@@ -15,6 +15,12 @@ of a the following OSTree specific properties:
 - ``branches`` - A list of branch names to be pulled during repository synchronization.
   When the value is ``nil`` (or not specified), all branches will be pulled.
 - ``gpg_keys`` - An (optional) list of GPG keys used to validate signed commits.
+- ``repair`` - An (optional) flag requesting the importer repair a corrupted :term:`local`
```

Comment:
On the server side, is there a difference between a flag and a boolean setting? An API user would have to still pass ``True`` to make it work. I think the "flag" concept might really come down to how it's exposed in pulp-admin.